backend/app/celery/resource_monitor.py

```
@@ -0,0 +1,194 @@
+# app/celery/resource_monitor.py
+import logging
+import psutil
+import threading
+import time
+from typing import List
```

Ruff (0.14.7)

`typing.List` is deprecated, use `list` instead

(UP035)